m4: do not force -I/usr/include in CPPFLAGS

This is so horribly broken for cross-compilation. :-(

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN openvmtools-9.4.6-1770165.orig/m4/vmtools.m4 openvmtools-9.4.6-1770165/m4/vmtools.m4
--- openvmtools-9.4.6-1770165.orig/m4/vmtools.m4	2014-07-02 00:21:14.000000000 +0200
+++ openvmtools-9.4.6-1770165/m4/vmtools.m4	2014-09-21 12:39:15.901659888 +0200
@@ -281,10 +281,10 @@
       if test "$os" = freebsd; then
          CUSTOM_$1_CPPFLAGS="-I/usr/local/include"
       else
-         CUSTOM_$1_CPPFLAGS="-I/usr/include"
+         CUSTOM_$1_CPPFLAGS=" "
       fi
       if test -n "$2"; then
-         CUSTOM_$1_CPPFLAGS="${CUSTOM_$1_CPPFLAGS}/$2"
+         : CUSTOM_$1_CPPFLAGS="${CUSTOM_$1_CPPFLAGS}/$2"
       fi
    fi
 ])
